Synview-backend/utils/JWTHelpers.ts

```diff
@@ -0,0 +1,41 @@
+import { Context } from "../deps.ts";
+export default async function Token(
+  context: Context,
+  next: () => Promise<unknown>
+) {
+  try {
+    const token = localStorage.getItem("token");
+    if (!token) {
+      return;
+    }
+    context.response.headers.set("Authorization", `Bearer ${token}`)
+    await next();
+  } catch (e) {
+    throw new Error("Didnt find any auth token: " + e);
+  }
+}
+export async function generateKey() {
+  return await crypto.subtle.generateKey(
+    { name: "HMAC", hash: "SHA-512" },
+    true,
+    ["sign", "verify"]
+  );
+}
```

Comment:
You are effectively generating a new key every time the server starts. This means that after a reboot all existing keys would become invalid, and it would not work in a multi-server scaled system either. 

The key should be part of the env so that it does not change across reboots. The repo I shared with you does this for example